cmd/ocic/container.go

```diff
@@ -10,10 +10,10 @@ import (
 
 	"github.com/urfave/cli"
 	"golang.org/x/net/context"
+	remocommandconsts "k8s.io/apimachinery/pkg/util/remotecommand"
```

Comment:
it's copy-pasted from kube, let's follow up on this so I don't have to rebase #511 again :)